### PR TITLE
Add Exa Wallet - Monero by @kristiinara

### PR DESCRIPTION
**Pull request by kristiinara**
1. [x] Project URL: https://github.com/exantech/exa-wallet-ios
2. [x] Update contents.json instead of README 
3. [x] One project per pull request
4. [x] Screenshot included 
5. [x] Avoid iOS or open-source in description as it is assumed
6. [x] Use this commit title format if applicable: Add app-name by @github-username
7. [x] Use approved format for your entry

### CODE REVIEW

File: contents.json [hunk 1]
```diff
@@ -18337,6 +18337,24 @@
...
+    },
+    {
+      "title": "Exa Wallet - Monero",
+      "category-ids": ["cryptocurrency"],
+      "description": "Receive, spend and store your Monero with an open-source multisignature wallet.",
+      "source": "https://github.com/exantech/exa-wallet-ios",
+      "homepage": "https://wallet.exan.tech/",
+      "itunes": "https://apps.apple.com/app/exa-wallet-monero/id1457024905
+      "license": "other",
+      "lang": "eng",
```
Comment (dkhamsing) [inline contents.json]: Please remove lang, it is only useful if it’s not English

File: contents.json [hunk 2]
```diff
@@ -18337,6 +18337,24 @@
...
       "suggested_by": "@lucanaef",
       "stars": 0
+    },
+    {
+      "title": "Exa Wallet - Monero",
+      "category-ids": ["cryptocurrency"],
+      "description": "Receive, spend and store your Monero with an open-source multisignature wallet.",
+      "source": "https://github.com/exantech/exa-wallet-ios",
+      "homepage": "https://wallet.exan.tech/",
+      "itunes": "https://apps.apple.com/app/exa-wallet-monero/id1457024905
```
Comment (dkhamsing) [inline contents.json]: You are missing the quote and comma at the end, could you add it please

File: contents.json [hunk 3]
```diff
@@ -18337,6 +18337,24 @@
       "date_added": "Oct 26 2019",
       "suggested_by": "@lucanaef",
       "stars": 0
+    },
+    {
+      "title": "Exa Wallet - Monero",
+      "category-ids": ["cryptocurrency"],
+      "description": "Receive, spend and store your Monero with an open-source multisignature wallet.",
```
Comment (dkhamsing) [inline contents.json]: Could you remove the period after wallet? We do not generally need it on this page. Thx

**Comment by dkhamsing** [comment]
This branch has conflicts, please resolve. Thx!

**Comment by kristiinara** [comment]
Resolved the conflict and made the requested changes.

**Comment by dkhamsing** [comment]
Thx!

**Pull request merged**